### ISSUE TITLE
1.4.8 Visual Presentation missing from WCAG 2.1 drafts

### ISSUE THREAD

**Issue by steverep** [w3c/wcag21#291]
[Visual Presentation (1.4.8)](https://rawgit.com/w3c/wcag21/resize-content_ISSUE-77/guidelines/#visual-presentation) is mysteriously missing all of its bullets in the 2.1 drafts.  I've brought this up in several emails weeks ago but it is still missing in the April 25 release.

**Comment by awkawk** [comment]
Steve, this was fixed by https://github.com/w3c/wcag21/commit/59291c7569b6c97567926d5af7e380f2f520e017
